steelthread-0/walkthrough.md

```diff
@@ -219,16 +219,30 @@ Name the integration ```steel thread```
 
 # Setup CHE to enable editing of the booster
 
-#TODO DAVE MARTIN
+## Self Signed Certs setup
```

Comment:
@maleck13 I've added some steps here.
I've deferred to the che docs for most of the meat, but I'm hopeful most of this is temporary.

For the self signed cert stuff, there may be a way in future to pass the cert onto a stack image at startup rather than a custom docker image with the cert baked in.
I can't see an easy way around the browser import of the cert though.

For the github step, if users authorized in Launcher first it shouldn't be needed in Che as well (card for this https://trello.com/c/DuRV9PUE/96-update-che-ansible-playbook-to-use-launchers-keycloak-instead-of-the-cluster-keycloak)